### ISSUE TITLE
HHL about optimizer

### ISSUE THREAD

**Issue by TJJTJJTJJ** [zhunzhong07/HHL#8]
Hi, I am sorry to bother you again.
In your HHL.py, there are some code, like 

```
        base_param_ids = set(map(id, model.module.base.parameters())) \
                         | set(map(id, model.module.triplet.parameters())) \
                         | set(map(id, model.module.feat.parameters())) \
                         | set(map(id, model.module.feat_bn.parameters()))

        new_params = [p for p in model.parameters() if
                      id(p) not in base_param_ids]
        param_groups = [
            {'params': model.module.base.parameters(), 'lr_mult': 0.1},
            {'params': new_params, 'lr_mult': 1.0}]
    optimizer = torch.optim.SGD(param_groups, lr=args.lr,
                                momentum=args.momentum,
                                weight_decay=args.weight_decay,
                                nesterov=True)
```

there is  confusion about the param_groups, after I test,  model.module.triplet.parameters(), model.module.feat.parameters(), model.module.feat_bn.parameters() are not included in param_groups, that means during the train process, the three parameters don't update? Or is it something I have ignored?

I have tried like:

```
        param_groups = [
            {'params': model.module.base.parameters(), 'lr_mult': 0.1},
            {'params': model.module.triplet.parameters(), 'lr_mult': 1.0},
            {'params': model.module.feat.parameters(), 'lr_mult': 1.0},
            {'params': model.module.feat_bn.parameters(), 'lr_mult': 1.0},
            {'params': new_params, 'lr_mult': 1.0}]
```
but the result is worse

Thank you

**Comment by zhunzhong07** [comment]
Hi, sorry for the late reply. In my experiment, I found it is better to fix these layers or give a low learning rate to them (triplet, feat, feat_bn). This could prevent overfitting on source data and achieve better results.

**Comment by TJJTJJTJJ** [comment]
yeah, and I have done some experiments, the results had proven your words.
